flight/Modules/VtolPathFollower/vtol_follower_control.c

```diff
@@ -522,82 +462,44 @@ int32_t vtol_follower_control_attitude(float dT)
 	float forward_accel_desired = -northCommand * cosf(yaw * DEG2RAD) + -eastCommand * sinf(yaw * DEG2RAD);
 	float right_accel_desired = -northCommand * sinf(yaw * DEG2RAD) + eastCommand * cosf(yaw * DEG2RAD);
 
+	StabilizationDesiredData stabDesired;
+
 	// Set the angle that would achieve the desired acceleration given the thrust is enough for a hover
 	stabDesired.Pitch = bound_min_max(RAD2DEG * atanf(forward_accel_desired / GRAVITY),
-	                   -guidanceSettings.MaxRollPitch, guidanceSettings.MaxRollPitch);
+	                   -guidanceSettings.MaxRollPitch, guidanceSettings.MaxRollPitch) + att_adj[1];
 	stabDesired.Roll = bound_min_max(RAD2DEG * atanf(right_accel_desired / GRAVITY),
-	                   -guidanceSettings.MaxRollPitch, guidanceSettings.MaxRollPitch);
+	                   -guidanceSettings.MaxRollPitch, guidanceSettings.MaxRollPitch) + att_adj[0];
```

Comment:
so here i think we should consider applying the max angles allows by the stabilization settings. by default these are greater than the navigation ones, but if nav is maxed at 20 and their range for att_adj is 35 (i still need to see that) they might get 55 deg which is well outside the default max stab settings.